storage/storage.go

```diff
@@ -230,6 +232,13 @@ func (storage DBStorage) Init() error {
 		storage.clustersLastChecked[clusterName] = lastChecked
 	}
 
+	if storage.dbDriverType == types.DBDriverSQLite3 {
+		_, err = storage.connection.Exec("PRAGMA foreign_keys = OFF;")
```

Comment:
We don't want to disable foreign_keys, they were enabled in helpers mock DB on purpose

Reply:
It's done because removing column in SQLite in migrations will not work, otherwise we need to recreate whole DB from scratch. Solutions that I see is or disable foreign keys for SQLite or remove SQLite support completely.

Reply:
Then you can turn them off temporarily in the migration.

> remove SQLite support completely.

I like this idea, but it's a decision we need to discuss as a team